README.md

```diff
@@ -71,9 +71,9 @@ There are no requirements regarding the manager account of the ASA, or the reser
             "type": "string",
             "description": "A URI pointing to a high resolution version of the asset's media."
         },
-        "attributes": {

```

Comment:
Could we keep this here, but move it to the bottom? In the description, we could say it's deprecated and should not be used for new NFTs, but that marketplaces need to support both to be fully compliant